spec/features/review_features/review_add_spec.rb

```diff
@@ -1,11 +1,20 @@
 require "rails_helper"
 
 feature "User visits the new review page" do
-  let (:wendys) { Restaurant.create(name: "Wendys", location: "Boston", dining_type: "Might be a selction", description: "Really long lines and really long text", hours: "10am-10pm") }
-  let (:baconator) { Burger.create name: "baconator", description: "lots of bacon", restaurant: wendys }
+  let (:wendys) { Restaurant.create(
```

Comment:
Follow the first comment, and fix this in all of your tests
"Avoid using {...} for multi-line blocks."
curly braces are for SINGLE LINE code, otherwise use "do" syntax I believe